docs/en/basic_configuration.md

```diff
@@ -73,6 +73,34 @@ action. This module provides the following engine:
 * `SiteTreePublishingEngine` - regenerates the static cache files after a page is published/unpublished. This makes use
 of both interfaces as object trigger refresh of other objects.
 
+#### Example:  Triggering a republish of the homepage after an update to a DataObject
+
+In order for the triggers to get picked up and executed, make sure to add the SiteTreePublishingEngine to your DataObject.
+```yaml
+Project\ORM\YourDataObject:
+  extensions:
+    - SilverStripe\StaticPublishQueue\Extension\Engine\SiteTreePublishingEngine
+```
+
+Then implement the StaticPublishingTrigger interface on your DataObject:
+
+```php
+class YourDataObject extends DataObject implements staticPublishingTrigger 
```

Comment:
Can you include the import statements in the example code, please.

Also, please use the correct case for the interface (`StaticPublishingTrigger`)